### ISSUE TITLE
Modification of User Interface.

### ISSUE THREAD

**Issue by r6654kumar** [r6654kumar/Interview-Voyage#2]

**Comment by alvarotorrestx** [comment]
Hey can I contribute to your project?

If so, what kind of UI updates are you looking for?

**Comment by r6654kumar** [comment]
Yes sure you can contribute to this project.
The UI should look more interactive....with proper navbar and footer...
For  now it just a simple UI.

**Comment by jenisesc** [comment]
Hey, id like to contribute to your project :)

**Comment by r6654kumar** [comment]
Sure

**Comment by jenisesc** [comment]
Please assign me the project thanks, could you provide more detail on what you're looking for?

**Comment by r6654kumar** [comment]
Assigned the project to you..
I am looking for a better user interface which should be interactive having modern components like navbar etc...

**Comment by jenisesc** [comment]
Hello, could you please provide me with a few details. Such as the features you want for the website? Currently, there's only posts about interviews and login/regsiter. Are there any other features you want included?

**Comment by r6654kumar** [comment]
Yes we can add more features such as secured login and registration using email or google sign in.
We can also add like dislike buttons to posts if possible and a better user interface.

**Comment by jenisesc** [comment]
Hi, i added some files of the new design, and created a pull request. Please let me know if you like it, or need anything else :)

**Comment by r6654kumar** [comment]
Hey...
The new design looks awesome. You may go ahead👍

**Comment by jenisesc** [comment]
Glad to be of help :)

**Comment by r6654kumar** [comment]
Hey @jenisesc 
Are you working on the designs provided by you..?
